packages/node_modules/@ciscospark/widget-message/src/container.js

```diff
@@ -247,9 +248,33 @@ export class MessageWidget extends Component {
       ) {
         // Send notification of new message
         const avatars = props.avatar.get('items');
+        let cardsLength = 0;
+        let message = lastActivityFromThis.object.displayName;
+
+        if (Object.prototype.hasOwnProperty.call(lastActivityFromThis, 'object')
```

Comment:
Using `Object.prototype.hasOwnProperty.call` is quite wordy and makes for difficult reading. Would it be better to shorten? to:
```js
if (lastActivityFromThis.object && lastActivityFromThis.object.cards && lastActivityFromThis.object.cards.length > 0)
```